docs/define/README.md

```diff
@@ -177,6 +179,20 @@ export default {
 }
 ```
 
+hTest can build the right checking function behind the scenes:
+
+```js
+import Color from "../src/index.js";
```

Comment:
Color.js is a very niche example. Check out the simple examples used by other testing frameworks